client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -169,18 +173,32 @@ const PackageInfo = ( props ) => {
 			</div>
 
 			<div>
-				<FormFieldset className="packages-step__package-weight">
-				<FormLabel htmlFor={ `weight_${ packageId }` }>{ translate( 'Total Weight' ) }</FormLabel>
-					<FormTextInputWithAffixes
-						id={ `weight_${ packageId }` }
-						placeholder={ translate( '0' ) }
-						value={ pckg.weight || '' }
-						onChange={ onWeightChange }
-						isError={ Boolean( pckgErrors.weight ) }
-						type="number"
-						noWrap
-						suffix={ weightUnit } />
-					{ pckgErrors.weight && <FieldError text={ pckgErrors.weight } /> }
+				<FormFieldset>
```

Comment:
I'm wondering if the weight and signature forms should be in the same `<fieldset>`? Or if we need `<fieldset>` at all, since we're not using `<legend>` in any case.

Reply:
Right, so after having consulted with resident a11y expert @ryelle, looks like we don't need to use `<fieldset>` at all here. This is because we're not trying to explain what each part of the form means, and we are not using `<legend>`, so it's safe to rely solely on the `<div>`s here.